### ISSUE TITLE
Inclusion of Anonymous Authentication 

### ISSUE THREAD

**Issue by richard-foxandcostudio** [the-road-to-react-with-firebase/react-firestore-authentication#7]
Can you provide some pointers on how to include anonymous authentication within this boilerplate?

**Comment by rwieruch** [comment]
I never used anonymous  authentication, so if you find out something about it, please leave a hint for others here :)

**Comment by richard-foxandcostudio** [comment]
I have managed to add anonymous authentication, but there is still work to add the ability to convert the anonymous account to a genuine account.  The code you use in the Account section for linking additional social logons should be easily adaptable, with the inclusion of the same process for e-mail authentication as well.  It is still very much work in progress though, but I hope to get it all neat and tidy and officially share it. I have also pulled in Foundations to provide some styling, though at present I have not improved things.

If you are interested in the work so far it is available here:
https://31xn0n8xv1.codesandbox.io/

I realise I am using forks where I should probably be using commits.  I am not a coder by trade and am new to CodeSandBox, React, GitHub, etc, so it is a lot to learn and it is slower going than I was expecting.  I suspect that once I have got it tidy I will need to raise a Pull Request from you and then I can, officially Fork it, integrate my changes and submit it back, so others can use it as a boiler plate? Am I on the right lines?
